src/components/dashboard/SendLinkSummary.js

```diff
@@ -85,6 +86,8 @@ const SendLinkSummary = ({ screenProps, styles }: AmountProps) => {
             status: 'pending',
             data: {
               counterPartyDisplayName,
+              senderEmail: email,
```

Comment:
were those fields names discussed & approved by @sirpy ?

Reply:
These are the ones specified in the task and had previously been used in the project.